backend/app.py

```diff
@@ -28,6 +28,24 @@ class Game(db.Model):
     def __repr__(self):
         return f"<Game {self.match_id}>"
 
+# Define a table to store players
+class Player(db.Model):
+    id = db.Column(db.Integer, primary_key=True)  # Unique player ID (auto-incremented)
+    player_id = db.Column(db.String, nullable=False)  # Player's unique ID
+    match_id = db.Column(db.String, db.ForeignKey('game.match_id'), nullable=False)  # Foreign key to Game's match_id
```

Comment:
The foreign key should go the other direction, from the match to the player id. We don't want to store the match id in the players column because a player can have many match ids. Instead, each match has one or more player ids. You should look up many to many relationships for how to handle that.